libs/sistema/src/lib/labels-expander/labels-expander.component.html

```diff
@@ -4,7 +4,7 @@
     <div class="label-selector">
       <pa-checkbox
         *ngFor="let label of labelset.value.labels"
-        [value]="currentSelection[labelset.key + '_' + label.title]"
+        [value]="currentSelection[getSelectionKey(labelset.key, label.title)]"
```

Comment:
it would be better to either create a small pipe for this, or to use directly the SEPARATOR variable you created. regular functions are called a lot during the change detection cycles, and performances can be impacted otherwise.

Reply:
Fixed